app/assets/stylesheets/team.css.scss

```diff
@@ -2,15 +2,14 @@
 // Team page
 //
 
-@import "./_variables";
-
-.team-section {
-    padding: 100px 0;
-    background: white;
-}
-
-.team-member {
-    padding-top: 30px;
+$black: #1E2028;
+$red: #DB2C38;
+$green: #83C057;
+$blue: #00A0BE;
+$past_cont_grey: #808080;
+
+.team-page-section {
+    padding: 50px 0;
```

Comment:
This one will need to be reduced to make up for the general application title padding.

Reply:
Updated